boxoffice/views/admin.py

```diff
@@ -24,11 +25,27 @@ def index():
     return dict(user=g.user)
 
 
+def jsonify_ic(ic):
+    if ic:
+        return {
+            'id': ic.id,
+            'name': ic.name,
+            'title': ic.title,
+            'url': '/ic/' + unicode(ic.id),
```

Comment:
Better to use `url_for`.

Reply:
Fixed

Reply:
We need relative path(without `/admin`) here since backbone is handling the routing